terraform/examples/alicloud-drds-instance/README.md

```diff
@@ -0,0 +1,17 @@
+### DRDS instance Example
```

Comment:
The directory terraform/example has been removed and please move the example into new directory examples.

Reply:
Done.